src/javaRosa.js

```diff
@@ -824,16 +824,17 @@ define([
         options.id = id;
 
         var widget = widgets.base(mug, options);
-        var $input = $("<textarea></textarea>")
+        var $input = $("<div contenteditable>")
```

Comment:
https://github.com/dimagi/Vellum/pull/432/files#diff-0c6adbb645e08e26dfcc96c660406ee4R1

I really like this solution. It will get a little more complicated once I add in support for HTML elements, but it's a good place to keep it.

I'm going to keep `val` `setValue` and `getValue` the same API wise. i.e you will still need to use <output value> when interacting with these. I'll probably create something like `getHTML` for tests and anything else that may need the HTML content, but I don't imagine that will be much.